apps/transport/client/javascripts/resource-viz.js

```diff
@@ -163,10 +164,50 @@ function fillGBFSMap (resourceUrl, fg, availableDocks, map, fitBounds = false) {
         .catch(e => removeViz(e))
 }
 
+// we want a custom message on the layers toggle control, depending on the GBFS vehicle type
+function setGBFSLayersControl (feeds, fg, availableDocks, map) {
+    if (!map.hasControlLayers) { // we don't want a new control at each data refresh
+        const labels = { bicycle: 'Vélos', car: 'Voitures', moped: 'Scooters', scooter: 'Trottinettes', other: 'Véhicules' } // according to GBFS v2.2
+        // 1 vehicle known vehicle type, we use it
+        // more vehicle types or unknown, we use a generic label : véhicules
+        getVehicleType(feeds).then(types => {
+            let vehicleLabel
+            if (types.length === 1) {
+                vehicleLabel = labels[types[0]]
+            } else {
+                vehicleLabel = 'Véhicules'
+            }
```

Comment:
```suggestion
            const vehicleLabel = types.length === 1 ? labels[types[0]] : 'Véhicules'
```
Si tu le sens (à tester, j'ai tapé ça dans GitHub)